docs/python/binary-extension.rst

```diff
@@ -351,14 +363,41 @@ The Python extension module (``.pyd`` file) depends directly on this library:
      Summary
    ...
 
-For built wheels, the dependency will be renamed from ``bezier.dll`` to a
-unique name containing the first 32 characters of a SHA256 hash (the hash
-input is ``bezier`` followed by the contents of the DLL file) to avoid a name
-collision and placed in the ``bezier.libs`` directory next to the ``bezier``
-package: for example
-``bezier.libs/bezier-40ff1ce7372f05ba11436ffbadd11324.dll``.
+and the local copy of ``libbezier`` depends on the other dependencies in
+``bezier.libs/`` (both directly and indirectly):
+
+.. testcode:: windows-dll
+   :hide:
+
+   site_packages_path = pathlib.Path(site_packages)
+   dll_path, = site_packages_path.glob("bezier.libs/bezier-*.dll")
+   dll_path = dll_path.relative_to(site_packages_path)
+   dll_path = os.path.join(os.pardir, str(dll_path))
+   invoke_shell("dumpbin", "/dependents", dll_path, cwd=base_dir)
 
-The ``libbezier`` DLL has **no external dependencies**, but does have
+.. testoutput:: windows-dll
+   :windows-only:
+
+   > dumpbin /dependents ..\bezier.libs\bezier-40ff1ce7372f05ba11436ffbadd11324.dll
+   Microsoft (R) COFF/PE Dumper Version ...
+   Copyright (C) Microsoft Corporation.  All rights reserved.
+
+
+   Dump of file ..\bezier.libs\bezier-40ff1ce7372f05ba11436ffbadd11324.dll
+
+   File Type: DLL
+
+     Image has the following dependencies:
+
+       libgcc_s_seh-1-5c71c85c0ca01174917203266ba98140.dll
+       libgfortran-5-08073c6868a1df2cbc5609e49cbe3ad8.dll
+       KERNEL32.dll
+       msvcrt.dll
```

Comment:
I have `13.1.0` on my dev machine (via MSYS2 + `pacman`) but GitHub Actions has `11.2.0.07112021` (via Chocolatey).

In addition to hash differences, this also produces **ordering** differences. In `13.1.0`, the order is:

```
       libgcc_s_seh-1-5c71c85c0ca01174917203266ba98140.dll
       KERNEL32.dll
       msvcrt.dll
       libgfortran-5-08073c6868a1df2cbc5609e49cbe3ad8.dll
```